featuretools/entityset/entityset.py

```diff
@@ -1151,10 +1152,35 @@ def add_last_time_indexes(self, updated_dataframes=None):
             self.dataframe_dict[df.ww.name] = df
 
         self.reset_data_description()
+
     # ###########################################################################
-    # #  Other ###############################################
+    # #  Pickling ###############################################
     # ###########################################################################
+    def __getstate__(self):
+        schemas = {}
+        for df_name, df in self.dataframe_dict.items():
+            try:
+                schema = df.ww.schema
+            except WoodworkNotInitError:
```

Comment:
Does this `WoodworkNotInitError` get thrown from doing `df.ww.schema` on a DataFrame that doesn't have Woodwork initialized?

 I think that the `schema` property is one of the only properties that shouldn't throw that error, so I'd expect that `df.ww.schema` would just be `None` if woodwork is not initalized. So if it's getting thrown, there might be some other woodwork attribute that's being referenced.

Then again, I'd also expect that all the dataframes in `dataframe_dict` would have woodwork initialized, so I very well could be misunderstanding the state of things when this method gets called.

Reply:
Yeah, I just removed those lines since 1) `dataframe_dict` should initialize woodwork and 2) you're right that the `schema` would just be `None`.